src/components/Bracket/Bracket.styles.ts

```diff
@@ -1,7 +1,7 @@
 import { cnb } from 'cnbuilder';
 
 // Bracket Curve styles
-export const root = 'su-border-current su-backface-hidden';
+export const root = 'children:su-border-current su-backface-hidden';
```

Comment:
Added this children: prefix to increase CSS specificity so the brackets can take on current text colors:
![Screenshot 2023-06-26 at 6 33 36 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/cab59e86-c99d-4fda-b36e-268017ca3841)

Likely I needed to add this update because when we modify the color theme from overriding to extend, the order of inherit and current text color has switched. Likely not a problem for other sites because this is the first site that I rely heavily on `currentColor` to also change border/background colors.